velox/functions/prestosql/tests/ArrayPositionTest.cpp

```diff
@@ -96,7 +132,48 @@ class ArrayPositionTest : public FunctionBaseTest {
       const int64_t instance,
       const std::vector<std::optional<int64_t>>& expected) {
     auto arrayVector = makeNullableArrayVector<T>(array);
+    auto size = arrayVector->size();
 
+    auto newSize = size * 2;
```

Comment:
Nit: These 2 functions share a lot of code , bar the instance vector and eval expr; maybe the 2 functions can be integrated ?

Reply:
Thank you for your comment! If you mean `testPositionDictEncodingWithInstance()` and `testPositionComplexTypeDictionaryEncodingWithInstance()`, this idea came to me as well. But the function for complex types requires constructing the array vector and search vector outside the function (because different complex types require different interfaces). This is not necessary for primitive types and I'd really like to keep the test cases of primitive types cleaner, so I separated these two functions.